src/org/traccar/Context.java

```diff
@@ -323,6 +325,9 @@ public static void init(String[] arguments) throws Exception {
         objectMapper = new ObjectMapper();
         objectMapper.setConfig(
                 objectMapper.getSerializationConfig().without(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS));
+        if (Context.getConfig().getBoolean("config.mapper.prettyPrintedJson")) {
```

Comment:
Please remove `config.` prefix from. It doesn't really make sense because all parameters are in "config".

Reply:
done